adhocracy-plus/config/settings/dev.py

```diff
@@ -54,3 +49,8 @@
     }
 
 CELERY_TASK_ALWAYS_EAGER = True
+
```

Comment:
probably good to add a comment here saying it should always be at the bottom

Reply:
done.